app/views/events/_form.html.erb

```diff
@@ -16,6 +16,18 @@
   <% end %>
 
   <div class="form-group">
+    <div class="btn-group col-lg-10 col-lg-offset-2" data-toggle="buttons">
+      <% Event.kinds.keys.each do |key| %>
+        <label class="btn btn-default <%= 'active' if @event.kind == key %>">
+          <%= f.radio_button :kind, key %>
+          <%= key.humanize %>
```

Comment:
Shouldn't the radio button be selected if the class 'active' is set for the label? Or am I missing something here?